apps/examples/protobuf_test/gtest_util_tests_others.cc

```diff
@@ -0,0 +1,28 @@
+/****************************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+// The following lines pull in the test files.
+#include "util/message_differencer_unittest.cc"
+#include "util/type_resolver_util_test.cc"
+
+extern "C"
+{
+    void gtest_util_tests_others_main(void)
+    {
+        return;
+    }
+}
```

Comment:
Above 5 files donot add any value to the test case. They are just dummy files. Their existence and removal both mean the same. They can be simply removed as they don't do anything

Reply:
The files are needed to pull the protobuf unittest files(like #include "proto3_lite_unittest.cc"). The including codes have to get together in gtest_main with other tests, but they issue build conflicts compiling with other test files so I let them in each separate file.